Adding issue template (#454)
flame-engine/flame: .github/issue_template.md

<!---

>  Please use this template to fill your issue, issues that does not
>  follow this will be asked to be edited to match the template, or will be closed

Reviewer: ```suggestion
 Please use this template to fill your issue. Issues that does not
 follow this will be asked to be edited to match the template, or will be closed.
```

Reviewer 2: Fill in*

>  We use github issues for problem/bug reports, for questions, please use the Discord channel, the link for it can be found on the project README

Reviewer: ```suggestion
 We use Github issues for problem/bug reports; for questions, please use the Discord channel, the link for which can be found on the project README.
```

Reviewer 2: Too many commas, split up in two sentences.

-->

## Description

Describe your issue here

## Development enviroment

Flame version:

<!--

> NOTE: Flame supports stable and beta channel of Flutter, issues out of those channels may not be prioritized, or closed.

Reviewer: ```suggestion
NOTE: Flame supports the stable and beta Flutter channels; issues outside of those channels may not be prioritized, or closed.
```

Reviewer 2: I would use full stop instead of semi-colon.

Reviewer: both work for me :)

-->

Flutter doctor:

```
 Include your flutter doctor output here
```

## Runtime

This issue is related to running on which platform? (Select all that apply)

 - [ ] Android :android:
 - [ ] iOS :apple:
 - [ ] Web :browser:

## Minimal reproducible code (Required for bugs)

> Include a minimal code that produces the problem, add it inline using code block markdown, or a link to github repository:

Reviewer: ```suggestion
Include a minimal code that produces the problem (add it inline using code block markdown), or a link to Github repository:
```

Reviewer 2: If we need to capitalize it correctly it should be GitHub ;)

> Include a minimal code that produces the problem, add it inline using code block markdown, or a link to github repository:

Reviewer 2: Produces -> shows/demonstrates
To a* github repository

Reviewer: I think he meant `reproduces`

Reviewer: also, `github` => `Github`

## Minimal reproducible code (Required for bugs)

> Include a minimal code that produces the problem (add it inline using code block markdown), or a link to Github repository:

Reviewer 2: Produces -> Reproduces
Since the part within the parenthesis is connected to the part after the comma either all of it should be within the parenthesis or none of it.

Reviewer 2: And if you want to capitalize github it should be GitHub.
